stress_test/controller.py

```diff
@@ -0,0 +1,349 @@
+# Copyright (c) 2016 Intracom S.A. Telecom Solutions. All rights reserved.
+#
+# This program and the accompanying materials are made available under the
+# terms of the Eclipse Public License v1.0 which accompanies this distribution,
+# and is available at http://www.eclipse.org/legal/epl-v10.html
+
+""" Controller Class- All controller-related functionality is here"""
+
+#import collections
+import json
+import common
+import logging
+import os
+import subprocess
+import time
+import util.netutil
+import util.process
+
+
+
+class Controller:
+
+
+    def __init__(self,ctrl_base_dir,test_config):
+
```

Comment:
create a new attribute (e.g. `base_dir`) to copy the contstructor parameter `ctrl_base_dir`.